README.md

```diff
@@ -1,11 +1,15 @@
-# HDC Flask DeID
+# Flask PHI ID
 A Flask-based webservice for identification of PHI in clinical narratives
 
 ##Installation Requirements
 You will need a Git Auth Token ( https://github.com/settings/tokens )stored in an environmental variable called 'HDCGITAUTHTOKEN'
 
 
-
+## General Installation
+venv venv
+source venv/bin/activate.sh
+python setup.py install
+python flaskphiid/run.py -e 0.0.0.0 -p 5000 #run flask app at endpoint 0.0.0.0 on port 5000
 
 ## test strings
```

Comment:
can I request a PHI worthy test string in the README?

note_text = 'Mr. Doe was diagnosed with Stage III adenocarcinoma of the left lung on 3/14/2018'
summin like that

Reply:
should already be one in there under the $ test strings heading